src/components/Layer/EditControl/EditControl.tsx

```diff
@@ -17,17 +14,12 @@ export function EditControl({
   layerState: LogicalLayerState;
   layerActions: LogicalLayerActions;
 }) {
+  const [featureFlags] = useAtom(featureFlagsAtom);
   async function editLayer() {
-    if (layerState.style?.type === 'mcda') {
-      const config = await editMCDAConfig(layerState);
-      if (config?.id) {
-        layerActions.destroy();
-        store.dispatch([
-          mcdaLayerAtom.createMCDALayer({ ...config, id: config.id }),
-          enabledLayersAtom.set(config.id),
-          ...getMutualExcludedActions(layerState),
-        ]);
-      }
+    if (layerState.style?.type === 'mcda' && featureFlags[FeatureFlag.MCDA]) {
+      import('~features/mcda').then(async ({ editMCDA }) => {
```

Comment:
а почему здесь лэйзи лоад юзаем?

Reply:
@IlyaLosikKontur 
Because importing editMCDA brings side-effect of calling `api/bavariate_matrix`, something I didn't quite expect until I encountered this.
`src/views/Map` imports features in similar way